src/types/component.ts

```diff
@@ -0,0 +1,33 @@
+// This needs to stay in sync with the TYPES enum
+// in planx-new/editor.planx.uk/src/@planx/components/types.ts
```

Comment:
I guess this means a next step would be to remove this from `planx-new`..? There's probably still a few duplicated types we can look to tidy up I'd guess.